features/sketch/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id(libs.plugins.wire.android.library.get().pluginId)
+    id("org.jetbrains.kotlin.android")
+    alias(libs.plugins.ksp)
+}
+
+android {
+    namespace = "com.wire.android.feature.sketch"
+}
+
+dependencies {
+    implementation(libs.androidx.core)
+    implementation(libs.androidx.appcompat)
+    implementation(libs.material)
+
+    val composeBom = platform(libs.compose.bom)
+    implementation(composeBom)
+
+    testImplementation(libs.junit4)
+    androidTestImplementation(libs.androidx.test.extJunit)
+    androidTestImplementation(libs.androidx.espresso.core)
```

Comment:
Maybe not a problem for dependencies, but we should try to centralise whatever we add to the `build.gradle.kts` files.

The script makes it easy to generate these, but it doesn't provide a solution to _maintain_ these generated files. It would be better to try to move as much as possible to the plugin instead, so when we change stuff it changes across all modules as well.

We could add `androidx.core`, `androidx.appcompat`, `junit4`, `androidx.test.extJunit` and `androidx.espresso.core` to the plugin, for example.
Even the Compose BOM, as the plugin is already enabling Compose anyway.

And maybe add a helper configuration to configure on each module, like so:

```kotlin
plugins {
    id(libs.plugins.wire.android.library.get().pluginId)
}

wireLibrary {
    compose = true
    supportRobolectric = false
}
```

Reply:
This is debatable and agree out of scope for this PR :D 

Anyway, I think going the extra mile is not needed in this case, we could have simple _dummy modules_, i.e. just Kotlin utility code, or a _Compose Components_ module, so I think the needs will be different for each of them in terms of dependencies, then the suggested dynamic config might be become handy.

For now, it is a problem we don't have (and probably never will) but just in case, later we can come back to this suggestion and go for it.